core/runtime/src/main/java/io/quarkus/runtime/Timing.java

```diff
@@ -66,12 +67,13 @@ public static void restart() {
         bootStartTime = System.nanoTime();
     }
 
-    public static void printStartupTime(String version, String features) {
+    public static void printStartupTime(String version, String features, String artifactId, String appVersion) {
         final long bootTimeNanoSeconds = System.nanoTime() - bootStartTime;
         final Logger logger = Logger.getLogger("io.quarkus");
         //Use a BigDecimal so we can render in seconds with 3 digits precision, as requested:
         final BigDecimal secondsRepresentation = convertToBigDecimalSeconds(bootTimeNanoSeconds);
-        logger.infof("Quarkus %s started in %ss. %s", version, secondsRepresentation, httpServerInfo);
+        logger.infof("%s %s (Running on Quarkus %s) started in %ss. %s", artifactId, appVersion, version,
```

Comment:
I think in this case the `build.finalName` value would be more interesting than the `project.artifactId`. It could fallback if necessary

Reply:
Also what happens if those values are unset? Perhaps it would be nice to still show the original message in this case

Reply:
> I think in this case the `build.finalName` value would be more interesting than the `project.artifactId`. It could fallback if necessary

How am I supposed to retrieve the `build.finalName`? 

> Also what happens if those values are unset? Perhaps it would be nice to still show the original message in this case

I will do.

Reply:
>How am I supposed to retrieve the build.finalName?
I think in a similar way as you retrieve the `artifactId`?

Reply:
I print the original message if version or artifact are not set but I have not included anything about the finalName, it is not clear from where I will retrieve it.

Reply:
@gastaldi can you unblock this?

Reply:
@FroMage I'm in vacation until June 24th and can have a look after that :smile: